blog/news-and-updates/2023-09-13-update.mdx

```diff
@@ -0,0 +1,56 @@
+---
+title: Developer weekly update September 13, 2023
+description: In this week's dev weekly update, we talk about the new 0.10.0 Motoko release, plus some new resources for developers on our dev docs. 
+tags: [Devs]
+image: /img/blog/dev-update-blog-sept-13.jpg
+---
+
+# Developer weekly update September 13, 2023
+
+![September 13 2023](../../static/img/blog/dev-update-blog-sept-13.jpg)
+
+This week, we have a new `moc` release, version `0.10.0`, which introduces the new stable regions feature, plus we'll talk about some new resources for developers on our dev docs. 
+
+## moc 0.10.0 - Motoko stable regions
+
+This past week, a new version of the Motoko compiler has been released - `0.10.0`. This release includes a new, highly anticipated feature: Motoko stable regions! This feature has been a long-running project that encompasses effort from several team members to continue developing Motoko to be a powerful, yet safe language, targeted toward ambitious developers and their projects. 
+
+Stable regions are a type of dynamically allocated, isolated, and independently growable regions of stable memory. Let's take a closer look at what this new stable regions feature is and how developers can use it.
+
+### What is a "region"? 
+
+A 'Region' is a new Motoko abstract data type that uses an experimental API which exposes low-level canister stable memory. `Region`s are unique since they use distinct portions of stable memory that never interfere with one another. New `Region`s can be created dynamically, and any old regions that are marked for garbage collection will be removed and reused for new `Region`s afterwards. 
```

Comment:
we may want to avoid calling this experimental, since the reason we developed stable regions to begin with was to solidify the previous "Experimental Stable Memory" we had for motoko